mmv1/products/dialogflow/api.yaml

```diff
@@ -334,58 +332,543 @@ objects:
+                        Configs for processing conversation.
+                      properties:
+                        - !ruby/object:Api::Type::Integer
+                          name: "recentSentencesCount"
+                          description: |
+                            Number of recent non-small-talk sentences to use as context for article and FAQ suggestion.
+              - !ruby/object:Api::Type::Boolean
+                name: "groupSuggestionResponses"
+                description: |
+                  If groupSuggestionResponses is false, and there are multiple featureConfigs in event based suggestion or StreamingAnalyzeContent, we will try to deliver suggestions to customers as soon as we get new suggestion. Different type of suggestions based on the same context will be in separate Pub/Sub event or StreamingAnalyzeContentResponse.
+                  If groupSuggestionResponses set to true. All the suggestions to the same participant based on the same context will be grouped into a single Pub/Sub event or StreamingAnalyzeContentResponse
+          - !ruby/object:Api::Type::NestedObject
+            name: "messageAnalysisConfig"
+            description: |
+              Configuration for message analysis.
+            properties:
+              - !ruby/object:Api::Type::Boolean
+                name: "enableEntityExtraction"
+                description: |
+                  Enable entity extraction in conversation messages on agent assist stage. If unspecified, defaults to false.
+                  Currently, this feature is not general available, please contact Google to get access.
+              - !ruby/object:Api::Type::Boolean
+                name: "enableSentimentAnalysis"
+                description: |
+                  Enable sentiment analysis in conversation messages on agent assist stage. If unspecified, defaults to false. Sentiment analysis inspects user input and identifies the prevailing subjective opinion, especially to determine a user's attitude as positive, negative, or neutral: https://cloud.google.com/natural-language/docs/basics#sentimentAnalysis For Participants.StreamingAnalyzeContent method, result will be in StreamingAnalyzeContentResponse.message.SentimentAnalysisResult. For Participants.AnalyzeContent method, result will be in AnalyzeContentResponse.message.SentimentAnalysisResult For Conversations.ListMessages method, result will be in ListMessagesResponse.messages.SentimentAnalysisResult If Pub/Sub notification is configured, result will be in ConversationEvent.new_message_payload.SentimentAnalysisResult.
+      - !ruby/object:Api::Type::NestedObject
+        name: "humanAgentHandoffConfig"
+        description: |
+          Configuration for connecting to a live agent.
+          Currently, this feature is not general available, please contact Google to get access.
+        properties:
+          - !ruby/object:Api::Type::NestedObject
+            name: "livePersonConfig"
```

Comment:
It seems that only of the `livePersonConfig` and `salesforceLiveAgentConfig` can be added. We may also need to add `conflicts` to these two fields.